config/samples/mellanox.com_v1alpha1_nicclusterpolicy.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     image: mofed
     repository: mellanox
     version: 5.5-1.0.3.2
+    terminationGracePeriodSeconds: 300
```

Comment:
Please, add the same for alm-examples in OCP bundle

Reply:
Done